Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -437,7 +437,8 @@ def prepare_ip_list_get_output(response: Dict) -> str:
             response.get("created_at")).strftime(HR_DATE_FORMAT),
         "Updated At": None if not response.get("updated_at") else arg_to_datetime(  # type: ignore
             response.get("updated_at", "")).strftime(HR_DATE_FORMAT),
-        "FQDNS": response.get('fqdns')[0].get('fqdn') if response.get('fqdns') else []
+        "IP Ranges": ", ".join([ip_list.get("from_ip") for ip_list in response.get("ip_ranges")]),
```

Comment:
We should include the `to_ip` field here as well. For example:

```
", ".join([ip_range.get("from_ip") + ("-" + ip_range["to_ip"] if ip_range.get("to_ip") else "") for ip_range in response.get("ip_ranges", [])])
```

Reply:
okay, will do the needful.

Reply:
Made the changes